src/components/Layout/Layout.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Layout from '.';
+
+describe('Layout', () => {
+  const children = <h1>hello</h1>;
+
+  beforeEach(() => {
+    render(<Layout>{children}</Layout>);
+  });
+
+  it('renders all children', () => {
+    const layout = screen.getByTestId('layout');
```

Comment:
Since the Layout has a styled component rendering a `<main>` HTML element, you should use `getByRole('main')` instead and leave test-id attributes for other elements difficult to reach